games/f.yaml

```
@@ -617,6 +617,26 @@
   type: remake
   updated: 2013-06-06
   url: http://freesiege.sourceforge.net/
+  
+- name: FreeSims
+  framework:
+  - MonoGame
+  images:
+  - http://i64.tinypic.com/2vuy42r.png
+  lang:
+  - C#
+  license:
+  - MPL
```

```suggestion
  - MPL
  content: commercial
```
i.e. the original game files are required

---

Thanks for suggestions, solved to get it done